src/data/list/forall2.lean

```diff
@@ -0,0 +1,237 @@
+/-
+Copyright (c) 2018 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro, Johannes Hölzl
+-/
+import data.list.basic
+import logic.relator
+import tactic.mk_iff_of_inductive_prop
+
+universes u v w z
+
+open nat function
+variables {α : Type u} {β : Type v} {γ : Type w} {δ : Type z}
+
+namespace list
+
+/- forall₂ -/
+
+section forall₂
```

Comment:
```suggestion
```

And move the `forall₂` inductive here as well.

Reply:
Moved.